src/backend/distributed/citus--7.5-3--7.5-4.sql

```diff
@@ -0,0 +1,47 @@
+/* citus--7.5-3--7.5-4 */
+
+CREATE FUNCTION pg_catalog.citus_query_stats(OUT queryid bigint,
+											 OUT userid oid,
+											 OUT dbid oid,
+											 OUT executor bigint,
+											 OUT partition_key text,
+											 OUT calls bigint)
+RETURNS SETOF record
+LANGUAGE C STRICT
+AS 'MODULE_PATHNAME', $$citus_query_stats$$;
+
+CREATE FUNCTION pg_catalog.citus_stat_statements_reset()
+RETURNS VOID
+LANGUAGE C STRICT
+AS 'MODULE_PATHNAME', $$citus_stat_statements_reset$$;
+
+CREATE FUNCTION pg_catalog.citus_stat_statements(OUT queryid bigint,
+												 OUT userid oid,
+												 OUT dbid oid,
+												 OUT query text,
+												 OUT executor bigint,
+												 OUT partition_key text,
+												 OUT calls bigint)
+RETURNS SETOF record
+LANGUAGE plpgsql
+AS $citus_stat_statements$
+BEGIN
+ IF EXISTS (
+ 	SELECT name FROM pg_extension
+ 	WHERE name = 'pg_stat_statements')
+ THEN
+ 	RETURN QUERY SELECT pss.queryid, pss.userid, pss.dbid, pss.query, cqs.executor,
+ 						cqs.partition_key, cqs.calls
+ 				 FROM pg_stat_statements(true) pss
+ 				 	JOIN citus_query_stats() cqs
```

Comment:
Slightly funny thing happens here when `pg_stat_statements` is loaded but completely empty (which it rarely is), because then citus_query_stats will never be called and we don't get an error. Seems ok though.